README.md

```diff
@@ -17,31 +17,36 @@
 
 ## Overview
 
-This is a [report processor](https://puppet.com/docs/puppet/7/reporting_write_processors.html) & fact terminus for Puppet to submit data to Splunk's logging system using Splunk's [HTTP Event Collector](https://docs.splunk.com/Documentation/Splunk/8.0.1/Data/UsetheHTTPEventCollector) service. There is a complimentary app in SplunkBase called [Puppet Report Viewer](https://splunkbase.splunk.com/app/4413/) that generates useful dashboards and makes searching this data easier. Please note that the Puppet Report Viewer app should be installed in Splunk before configuring this module.
+This module provides three services to Puppet and Splunk users.
 
-There is also the [Puppet Alert Actions](https://splunkbase.splunk.com/app/4928/) app, which contains the alert actions that were previously shipped in the Puppet Report Viewer:
+1. A report processor to allow sending Puppet Agent run reports to Splunk. When a Puppet agent completes a run and submits the results to Puppet, this module's processor can be invoked to send that report to Splunk. After this module is installed in your environment, to enable sending node reports to Splunk, do the following:
```

Comment:
I might say "some of the report data" since I don't think we are sending the entire report.